packages/block-editor/src/components/block-preview/index.js

```diff
@@ -44,6 +48,7 @@ export function BlockPreview( { blocks, viewportWidth = 700 } ) {
 			<AutoHeightBlockPreview
 				key={ recompute }
 				viewportWidth={ viewportWidth }
+				__experimentalPadding={ __experimentalPadding }
```

Comment:
There's not much information for me to work from on why we're introducing the pattern, aside from it being "a small issue". I'd personally prefer this to be done through stylesheets and not props passing / style props. Unclear if that's possible.

Reply:
Yes, I should have been more clear about this. It's not possible through CSS because of the "scaling" and because we want the padding to apply inside the container that apply the editor styles. If I just do it in CSS, the scaling breaks.